terraform/modules/grafana/variables.tf

```diff
@@ -20,6 +20,6 @@ variable "ip_allowlist" {
 
 variable "kms_key_id" {}
 
-variable "vpc_cidr_block" {}
-
 variable "vpc_id" {}
+
+variable "private_subnet_ids" {}
```

Comment:
Missing line break

Reply:
Sorry, I didn't check this one. I've added in the line break.